### PR TITLE
Add page on disks creation

### PR DESCRIPTION
**Pull request by ayaz91**

### CODE REVIEW

File: _sass/_base.scss [hunk 1]
```diff
@@ -15,3 +15,17 @@ $sidenav-width: 250px;
 	-moz-box-sizing: border-box;
 	box-sizing: border-box;
 }
+
+table{
```
Comment (powerjg) [inline _sass/_base.scss]: Why was this change needed? I'm surprised that the default table style doesn't work...

Comment (ayaz91) [reply _sass/_base.scss]: Without this change, the borders of the table do not appear on the web page

**Comment by powerjg** [comment]
manually rebased and merged.
